app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -104,13 +104,16 @@ open class HomeActivity : AppCompatActivity() {
 
     override fun onResume() {
         super.onResume()
-        CoroutineScope(Dispatchers.Main).launch {
-            // Make sure accountManager is initialized.
-            components.backgroundServices.accountManager.initAsync().await()
-            // If we're authenticated, kick-off a sync and a device state refresh.
-            components.backgroundServices.accountManager.authenticatedAccount()?.let {
-                components.backgroundServices.syncManager.syncNow(startup = true)
-                it.deviceConstellation().refreshDeviceStateAsync().await()
+
+        if (! this.isInExperiment(Experiments.asFeatureSyncDisabled)!!) {
+            CoroutineScope(Dispatchers.Main).launch {
+                // Make sure accountManager is initialized.
+                components.backgroundServices.accountManager.initAsync().await()
```

Comment:
@grigoryk besides adding this flag for `onResume`, is there a way we can "pause" sync if flag `asFeatureSyncDisabled` is `true`

Reply:
ping @grigoryk